packages/core/src/dialects/abstract/query-generator.types.ts

```diff
@@ -4,3 +4,8 @@ export interface QueryWithBindParams {
   query: string;
   bind: BindOrReplacements;
 }
+
+export interface TruncateTableOptions {
```

Comment:
```suggestion
export interface TruncateTableQueryOptions {
```

We do this for the other query generator method as well, as not to interfere with possible query interface method with the same name

Reply:
I have made suggestions for refactoring in the abstract folder, and I think you'll manage to do the same for the other dialect folders

Reply:
Done